requirements.txt

```diff
@@ -7,5 +7,5 @@ pylru==1.0.6
 git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
 PyContracts==1.7.1
 underscore.py==0.1.6
-pynliner==0.5.2
+pynliner==0.8.0
```

Comment:
What is reason behind this version upgrade?

Reply:
`pynliner==0.5.2` has a dependency `beautifulsoup`. `beautifulsoup3` doesn't support python 3.
`pynliner==0.8.0` used `beautifulsoup4` which supports both python3 and python2.

Reply:
that make sense